constants.py

```diff
@@ -25,6 +25,7 @@
 
 # Resource types.
 RESOURCE_TYPE_NETWORK_POLICY = "NetworkPolicy"
+RESOURCE_TYPE_EGRESS_POLICY = "Egresspolicy"
```

Comment:
@caseydavenport No idea :( But I think the name is related to how you create this TPR itself, so we can call it as CalicoPolicy or even NetworkPolicy, that the API path changes with the TPR definition. 

Taking a look, you can see in this samefile that the 'THIRDPARTY_API' is %s/apis/projectcalico.org. This is defined in the TPR resource [here](https://github.com/estaleiro/k8s-policy/blob/8bdfb539d36d3d99927bf9d2e1c81b6aff4376fc/examples/egress/egress-resource.yaml#L4)

I've based this idea in how Voyager Ingress achieve / extends the Ingress object (https://github.com/appscode/voyager/blob/master/docs/user-guide/README.md), but I see that you still have to specify in your spec the apiVersion that should be used, otherwise it will use the default NetworkPolicy object.